src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Services/Interfaces/IEnterpriseUserAuthenticationService.cs

```diff
@@ -0,0 +1,19 @@
+using System.Net.Http;
+using System.Threading.Tasks;
+using Altinn.Platform.Authentication.Model;
+
+namespace Altinn.Platform.Authentication.Services.Interfaces
+{
+    /// <summary>
+    /// Interface describing methods related to authentication of enterprise users.
+    /// </summary>
+    public interface IEnterpriseUserAuthenticationService
+    {
+        /// <summary>
+        /// Method that fetches a response from authentication on sblbridge
+        /// </summary>
+        /// <param name="credentials">Credentials for an enterpriseuser</param>
+        /// <returns>User profile connected to given credentials</returns>
```

Comment:
return message doesn't match the return type. Could the comment be more specified?

Reply:
Changed the comment to specify that the method returns a UserAuthenticationResult.